doc/modules/ROOT/pages/common-datasets.adoc

```diff
@@ -92,6 +92,38 @@ The default is to load IMDB dataset as `undirected = True`. If loaded as directe
 |===
 
 
+=== LastFM
+
+A heterogeneous graph that is used to benchmark link prediction models, for example used by https://arxiv.org/pdf/2002.01680.pdf[MAGNN: Metapath Aggregated Graph Neural Network for
```

Comment:
Nitpick: I think we've agreed in the past to not add links to papers since they could change, and only refer to publications by name